mappings/net/minecraft/util/annotation/DeobfuscateClass.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_6177 net/minecraft/util/annotation/DeobfuscateClass
+	COMMENT An annotation on classes. When a class is annotated, the class itself and its
+	COMMENT fields and methods are not obfuscated. Mainly used by blaze3d.
```

Comment:
This is probably a blaze3d variant of `DontObfuscate` in Mojang's names, but we can call it this IMO.

Reply:
Not sure; this one is actually always used on classes while the obf one is on methods.